src/blockchain/ain/jellyfish/jellyfish.service.ts

```diff
@@ -183,20 +223,145 @@ export class JellyfishService {
     return this.createTxAndCalcFee(utxo, vins, vouts, witnesses);
   }
 
+  private async generateRawTxForSendAccount(
+    from: string,
+    to: string,
+    token: number,
+    amount: BigNumber,
+  ): Promise<RawTxDto> {
+    const [fromScript, fromPubKeyHash] = RawTxUtil.parseAddress(from);
+    const [toScript] = RawTxUtil.parseAddress(to);
+
+    const utxo = await this.utxoProvider.provideNumber(from, 1, UtxoSizePriority.BIG);
```

Comment:
Why `UtxoSizePriority.BIG`?

Reply:
good catch. that function was written before .FITTING existed and didn't see the wrong size priority. thanks